package.json

```diff
@@ -61,7 +61,7 @@
     "@storybook/addons": "^5.2.1",
     "@storybook/react": "^5.2.1",
     "@storybook/storybook-deployer": "^2.8.1",
-    "autoprefixer": "^9.5.0",
+    "autoprefixer": "^9.7.1",
```

Comment:
why these updated?

Reply:
Did npm update when my shell requested it.